src/scss/_theme.scss

```diff
@@ -33,26 +32,26 @@ $input-padding: $spacing-base * 1.5;
 $boolean-padding: $spacing-double;
 
 :root, .apos-theme-light {
-  --primary: #6516DD;
```

Comment:
What is the logic behind giving these vars *more* specific convention? Wouldn't 'danger', for example, have utility in multiple contexts (inputs, notifications)?

Also -- how do we feel about HSL? It's more human-readable than HEX. Figma component library surfaces both HSL and HEX.

Reply:
Was thinking about this .. `primary-button` should probably flip back to `primary` as a base, the same was confirm, danger, etc are base colors that are shared.
But the state variations of it are tied to user interaction, so it could make sense to leave them like `primary-button-hover` etc ..

Reply:
Flipped this back to `primary`